stingray/powerspectrum.py

```diff
@@ -272,9 +279,11 @@ def rebin(self, df, method="mean"):
         """
 
         ## rebin power spectrum to new resolution
-        binfreq, binps, step_size = utils.rebin_data(self.freq[1:],
-                                                     self.ps[1:], df,
-                                                     method=method)
+        ## an empty variable is required to hold the yerr value returned
+        binfreq, binps, _, step_size = utils.rebin_data(self.freq[1:],
+                                                        self.ps[1:],
+                                                        self.ps_err[1:], df,
+                                                        method=method)
 
```

Comment:
I'm now confused on what is supposed to be right behavior. I had to change the test function `rebin_data`to pass the tests with the changes tha you suggested.

Can you please check the latest build?